Source/Core/Plugins/EventProcessor/Default/20_ErrorPlugin.cs

```diff
@@ -19,6 +19,9 @@ public class ErrorPlugin : EventProcessorPluginBase {
 
             if (String.IsNullOrWhiteSpace(context.Event.Message))
                 context.Event.Message = error.Message;
+            
+            if (context.StackSignatureData.Count > 0)
+                return Task.CompletedTask;
```

Comment:
@ejsmith is there ever a time where we would want to run stacking data and set the target info if you have a manual stack signature?